content/concept-freeze.md

```diff
@@ -6,7 +6,9 @@ The XRP Ledger gives addresses the ability to freeze non-XRP balances, which can
 * [**Global Freeze**](#global-freeze) - Freeze all counterparties.
 * [**No Freeze**](#no-freeze) - Permanently give up the ability to freeze individual counterparties, as well as the ability to end a global freeze.
 
-Because no party has a privileged place in the XRP Ledger, the freeze feature cannot prevent a counterparty from conducting transactions in XRP or funds issued by other counterparties. No one can freeze XRP.
+Because no party has a privileged place in the XRP Ledger, the freeze feature cannot prevent a counterparty from conducting transactions in XRP or funds issued by other counterparties.
+
+**Tip:** The freeze feature only applies to issued currencies. No one can freeze XRP.
```

Comment:
I'm fine with this, but at your discretion, consider rendering "No one can freeze XRP" in *bold*.